### PR TITLE
added the myplace details

### PR DESCRIPTION
**Pull request by sumanth51**
review this @susesi

### CODE REVIEW

**Comment by susesi** [comment]
Add Vizag items too

**Comment by hcanna** [comment]
@sumanth51 ... Were you able to add a commit from the command line?

File: hometown51/myplace.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## places to visit
+
+- Charminar 
+- birla temple
```
Comment (susesi) [inline hometown51/myplace.md]: Add Oldtown too

**Comment by sumanth51** [comment]
yes i am able to comment @hcanna

**Comment by sumanth51** [comment]
This closes #14

**Pull request merged**